cmd/bee/cmd/db.go

```diff
@@ -230,3 +230,24 @@ func dbNukeCmd(cmd *cobra.Command) {
 	c.Flags().Duration(optionNameSleepAfter, time.Duration(0), "time to sleep after the operation finished")
 	cmd.AddCommand(c)
 }
+
+func removeContent(path string) error {
+	dir, err := os.Open(path)
```

Comment:
this is a bit misleading. i believe that if you use this you would then get everything that sits in the directory in which `path` is contained. did you try this manually? i would consider using [filepath.Walk](https://pkg.go.dev/path/filepath#Walk)

Reply:
yeah, tried manually, root folder is preserved